sidebars.js

```diff
@@ -95,6 +95,7 @@ const sidebars = {
         'en/integrations/data-ingestion/kafka/index',
         'en/integrations/data-ingestion/etl-tools/dbt/index',
         'en/integrations/data-ingestion/redshift/index',
+        'en/integrations/deployment/easypanel/index',
```

Comment:
We keep this list minimalistic not to overload the menu.
```suggestion
```

Reply:
I hid it under the sub menu.

Reply:
@mshustov I hid it under the sub menu.